apps/cli/src/commands/signMessage.ts

```diff
@@ -1,13 +1,14 @@
 import fs from "fs";
 import { from } from "rxjs";
-import { mergeMap } from "rxjs/operators";
+import { switchMap } from "rxjs/operators";
 import { withDevice } from "@ledgerhq/live-common/hw/deviceAccess";
-import signMessage from "@ledgerhq/live-common/hw/signMessage/index";
-import { currencyOpt, inferCurrency } from "../scan";
+import signMessage, { prepareMessageToSign } from "@ledgerhq/live-common/hw/signMessage/index";
+import { scan, scanCommonOpts } from "../scan";
+
 export default {
```

Comment:
Just taking the occasion to express a personal preference: I think it's best to use named exports everywhere. It offers better ease of search and guarantees the name is the same everywhere we import. With default exports we can have different names for every import

```ts
// fileA.ts
import SignMessageCommand from "./signMessage";
// fileB.ts
import CommandSignMessage from "./signMessage";
// fileC.ts
import SignMessage from "./signMessage"
```

Not very relevant in the CLI but mostly mostly for the libs functions and ui components

Reply:
I totally agree :)